src/App.tsx

```diff
@@ -4,67 +4,114 @@ import Signup from './modules/Signup';
 import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
 import ViewProducts from './pages/ViewProducts';
 import AdminDashBoard from './pages/AdminPage';
-import  Home  from './modules/Home';
-import ViewUsers from './pages/ViewUsers';
+import ViewUsers from '././pages/ViewUsers';
 import ViewOrders from './pages/ViewOrders';
 import UserPage from './pages/UserPage';
-import BuyProducts from './pages/BuyProduct';
+import BuyProducts from './pages/BuyProductPage';
 import Error from './modules/Errror';
+import UserOrders from './pages/UserOrders';
+import { Provider } from 'react-redux';
+import store from './api/store';
+import Protected from './modules/AuthLayout';
+import Landing from './pages/LandingPage';
+import Cart from './pages/CartPage';
+import Orderconfirm from './modules/Orderconfirm';
+import Footer from './pages/Footer';
+import { ToastContainer } from 'react-toastify';
+import "react-toastify/dist/ReactToastify.css";
 
-const queryClient=new QueryClient();
+
+
+const queryClient = new QueryClient();
 
 function App() {
     const router = createBrowserRouter([
         {
             path: "/signup",
-            element: <Signup />
+            element:
+                <Protected authentication={false}>
+                    <Signup />
+                </Protected>
         },
         {
             path: "/login",
```

Comment:
Create a file called routes and store all this routes there & then import here eg instead of using "/login"
You can store it inside the ROUTES object & use it here as ROUTES.LOGIN